encoding/tests/maccommands.rs

```diff
@@ -328,6 +328,18 @@ fn test_channel_mask() {
     assert_eq!(&chan_mask.unwrap().statuses::<16>()[..], &expected[..]);
 }
 
+#[test]
+fn test_channel_mask_enable_and_disable_channel() {
+    let data = vec![0x00];
+    let mut chan_mask = ChannelMask::<1>::new(&data[..]).unwrap();
+    chan_mask.set_channel(7, true);
+    let expected = vec![false,false,false,false,false,false,false,true,];
+    assert_eq!(&chan_mask.statuses::<8>()[..], &expected[..]);
+    chan_mask.set_channel(7, false);
+    let expected = vec![false,false,false,false,false,false,false,false,];
+    assert_eq!(&chan_mask.statuses::<8>()[..], &expected[..]);
+}
```

Comment:
Maybe it would be better to have a test with byte index > 0 (probably using something like `let expected = [false; 16]; expected[11] = true;` for example).

Reply:
That is better. Fixed